src/llrt-function.ts

```
@@ -46,6 +46,14 @@ export interface LlrtFunctionProps extends NodejsFunctionProps {
    * @default - If this option is not provided, the LLRT binary is downloaded from GitHub and cached in the .tmp directory.
    */
   readonly llrtBinaryPath?: string;
+
+  /**
+   * If `true` then the LLRT runtime will be built in a layer that can be shared amongst
+   * other `LLrtFunction`s that utilise the same `LlrtBinaryType` (this property will be ignored if `llrtBinaryPath` is set).
```

Instead of ignoring, let's throw a validation error.